### ISSUE TITLE
Story: Forecasting: Skills, Summer & Sustainability

### ISSUE THREAD

**Issue by z-schwartz** [IITIDIDX597/F2021_Team2#21]
**PROJECT GOAL**
Community Catalyst connects a resident in the community with partners that help transform the collective vision for the neighborhood into action ensuring a more equitable and sustained community recovery.

**THE REQUISITE SUB-HILL STATEMENT**
Grouping: Forecasting
1. Volunteers can see daily, weekly and monthly progress on the dashboard that helps them plan and schedule sign ups for new programs
2. With the help of a futurecast insight tool based on their previous experiences and current program progress, volunteers can see new program suggestions to choose from: E.g. "If I do this, then x,y,z."

**THE STORY FOR REVISION**
**As a :** volunteer

**I want:** to learn skills that can help me contribute to the environmental sustainability of my community, during summer. 

**So that:** I can add value to my community

**Comment by z-schwartz** [comment]
- [x] @ronjmartin - as discussed, please refine 2 more times in the conversation space of the GitHub before our 5 o'clock meeting.

**Comment by F2021MartinRon** [comment]
**Iteration 1**
**As a:** Volunteer
**I want:** to easily schedule my volunteering efforts based on my free time  
**So that:** I can maximize the amount of time I am contributing to sustainable community leadership

**Comment by F2021MartinRon** [comment]
**Iteration 2**
**As a:** Volunteer
**I want:** to access a scheduling tool on the dashboard 
**So that:** I can efficiently spend my free time volunteering on the projects I prefer

**Comment by akallish** [comment]
> **Iteration 2**
> **As a:** Volunteer
> **I want:** to access a scheduling tool on the dashboard
> **So that:** I can efficiently spend my free time volunteering on the projects I prefer

@z-schwartz this may be the one

**Comment by z-schwartz** [comment]
> > **Iteration 2**
> > **As a:** Volunteer
> > **I want:** to access a scheduling tool on the dashboard
> > **So that:** I can efficiently spend my free time volunteering on the projects I prefer
> 
> @z-schwartz this may be the one

Yes. This is the one

**Comment by akallish** [comment]
> Yes. This is the one

@z-schwartz this story is good

**Comment by Tosgirl** [comment]
@Tosgirl iteration 1, @UjjwalAS iteration 2

**Comment by Tosgirl** [comment]
SELECTED STORY
> **Iteration 2**
> **As a:** Volunteer
> **I want:** to access a scheduling tool on the dashboard
> **So that:** I can efficiently spend my free time volunteering on the projects I prefer

ACCEPTANCE CRITERIA ITERATION 1
Given: I login to my dashboard and go to my calendar tool
When: I click on different days of the week to schedule my free time
Then: I can plan and commit more time to working efficiently on my favorite projects

**Comment by Tosgirl** [comment]
@UjjwalAS you got this

**Comment by UjjwalAS** [comment]
ACCEPTANCE CRITERIA ITERATION 2

Given: That I access calendar dashboard on my Community Catalyst application and my CC calendar automatically integrates with my google calendar ( or other calendar tools that I use) to project free time slots in the week

When: I click on the free time slots projected by the application

Then: the screen shows me available volunteering opportunities with details ( such as who is partner and what is the cause, who else is volunteering, for how long will the opportunity require my commitment) during those time slots for selection, so that I can pick and choose or reject volunteering opportunities as per my preferences.


@Tosgirl @akallish 

I have made a few changes to the acceptance criteria based on the following:

1. I believe the application needs to consider other time management tools outside CC application system to efficiently manage volunteer's free time.

2. It should reduce or completely eliminated cognitive load on user's end while he is trying to match his free time with volunteering opportunities of his choice- (automatic suggestions of free time overlapped with available volunteering opportunities during those times would do that- as user can accept or reject with just a click)

Let me know what do you think.

**Comment by akallish** [comment]
@UjjwalAS @Tosgirl @z-schwartz good conversation. This platform may pull data from other digital systems that a volunteer uses like calendar, LinkedIn, et. That helps cc find patterns and make recommendations

**Comment by UjjwalAS** [comment]
@akallish yes. I think that will make recommendations more accurate and easy for the user

**Comment by UjjwalAS** [comment]
**ACCEPTANCE CRITERIA ITERATION 3**

Given: That I am on the calendar section on my dashboard and I see an integrated schedule that combines my free time with volunteering project time slots 

When: I click on the free time slots projected by the application

Then: the screen shows me available volunteering opportunities with details: 
1. Who is partner
2. What is the cause, 
3. Who else is volunteering, 
4. Duration of the commitment
5. Eligibility criteria

so that I can pick and choose or reject volunteering opportunities as per my preferences.

**Comment by z-schwartz** [comment]
Then: cc provides relevant details to my volunteer profile as well as specific event/logistical details.

**Comment by z-schwartz** [comment]
### **This looks good.** 

**Given:** That I am on the calendar section on my dashboard and I see an integrated schedule that combines my free time with volunteering project time slots

**When:** I click on the free time slots projected by the application

**Then:** cc provides relevant details to my volunteer profile as well as specific event/logistical details.

- Who is partner
- What is the cause,
- Who else is volunteering,
- Duration of the commitment
- Eligibility criteria

so that I can pick and choose or reject volunteering opportunities as per my preferences.

**Comment by akallish** [comment]
Good details
